main-app/app/scripts/controllers/game-controller.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+    'use strict';
+    angular.module('Tombola.NoughtsAndCrosses')
+        .controller('MainController', ['$scope', '$location', 'GameModel', 'Audio', function ($scope, $location, GameModel, audio) {
+
+            $scope.model = GameModel;
+            var numThemes = 3;
+
+            $scope.currentTheme = 1; //standard theme
+
+            $scope.changeTheme = function(){
```

Comment:
Again, you could slim down the controller by creating a theme switching provider.